src/utils/history.js

```diff
@@ -0,0 +1,3 @@
+import { createBrowserHistory } from "history";
+
+export default createBrowserHistory();
```

Comment:
1. Can this be a typescript file?
2. Can we rename this folder to not be "utils"? In my experience, the `utils` folder eventually just becomes a big list of items that have about as much organization as its parent folder. Prefer something like `src/history` or `src/entry-point-client`.

Reply:
Both done!